dll/win32/oleaut32/msvc.S

```diff
@@ -13,9 +13,42 @@
 .code64
 
 PUBLIC call_method
-call_method:
-    int 2ch
+FUNC call_method
+    push rbp
+    .PUSHREG rbp
+    mov rbp, rsp
+    push rsi
+    .PUSHREG rsi
+    push rdi
+    .PUSHREG rdi
+    .ENDPROLOG
+
+    mov rax, rcx
+    mov rcx, 4
+    cmp rdx, rcx
+    cmovg rcx, rdx
+    lea rdx, [rcx * 8]
+    sub rsp, rdx
```

Comment:
Usually this requires a call to __chkstk. I guess we can live without, if we assume that this function can never be called with any parameter count that makes the total stack allocation exceed 4k.